<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesFieldWizard</class>
 <widget class="QWidget" name="FilesFieldWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>3. Configure field properties</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Visible sections</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>File list fields display the file icon and file name as default sections.&lt;br&gt;
Additional sections can be configured:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fileTypeCheckBox">
        <property name="toolTip">
         <string>File extension such as .pdf</string>
        </property>
        <property name="text">
         <string>Show file type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dateCheckBox">
        <property name="toolTip">
         <string>date and time as the file was imported</string>
        </property>
        <property name="text">
         <string>Show date and time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="originalPathCheckBox">
        <property name="toolTip">
         <string>save the original directory path from which a file was imported</string>
        </property>
        <property name="text">
         <string>Show originl import path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>&amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>&amp;Finish</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
